src/photoshop/OpenColorIO_PS_Context.cpp

```diff
@@ -283,3 +283,19 @@ OpenColorIO_PS_Context::getDefaultTransform(const std::string &device) const
     return _config->getDefaultView( device.c_str() );
 }
 
+
+void
+OpenColorIO_PS_Context::getenv(const char *name, std::string &value)
+{
+#ifdef __APPLE__
```

Comment:
What about the linux support ?

Reply:
Photoshop and After Effects are Mac and Windows only.